api/rest/tracer.go

```diff
@@ -31,6 +31,26 @@ func AddTracers(w http.ResponseWriter, r *http.Request) {
 	w.Write(ret)
 }
 
+// AddTracers handles the HTTP API request to add a set of tracers from a Request
+// to the database.
+func updateTracer(w http.ResponseWriter, r *http.Request) {
+	fmt.Println("HTITHTITHTIHTI")
```

Comment:
I think we need to change this method signature to types.Tracers rather than types.Request since we don't have most of the Request data types when we do the update. We are usually just wanting to update the Tracer with ID blah. See 
https://github.com/nccgroup/tracy/blob/master/plugin/scripts/highlight.js#L207 for how we invoke the API in the plugin. I think we can just merge it with master: https://github.com/nccgroup/tracy/blob/master/api/rest/tracer.go#L17